### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/frontmatter/home.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#592] [Announcements]
# https://theme-hope.vuejs.press/zh/config/frontmatter/home.html

home 类型: boolean; 默认值: false; 设置为 true 时启用首页样式 title 类型: string; 必填: 否; 设置页面标题，会用于路径导航、页面增强等。 heroText 类型: string | false; 默认值: 

https://theme-hope.vuejs.press/zh/config/frontmatter/home.html

**Comment by PiniaTop** [comment]
bgImageDark ，这个属性设置后，切换暗黑模式不生效
